CodeQL.yml

```diff
@@ -0,0 +1,4 @@
+path_classifiers:

```

Comment:
I am slightly confused about the guidance shared by codeql team - https://eng.ms/docs/cloud-ai-platform/devdiv/one-engineering-system-1es/1es-docs/codeql/troubleshooting/bugs/generated-library-code

It says that for the repos onboarded to 1ES, this approach of codeql.yml won't work and there is another approach to exclude tests projects -

![image](https://github.com/user-attachments/assets/26d46a52-b5b1-4283-b9e8-1db15b38d4a1)


For 1ES templates - https://eng.ms/docs/cloud-ai-platform/devdiv/one-engineering-system-1es/1es-docs/codeql/1es-codeql#additional-options

![image](https://github.com/user-attachments/assets/d1c41650-e4d9-4578-a335-f06752f9761e)

Do we know which of these approaches are correct for our repo with 1ES pipelines?

Reply:
Agreed, Changed the implementation